.eslintrc.json

```diff
@@ -8,7 +8,25 @@
     "react/prop-types": 0,
     "react/no-unknown-property": ["error", { "ignore": ["sx"] }],
     "no-unused-expressions": "off",
-    "@babel/no-unused-expressions": "warn"
+    "@babel/no-unused-expressions": "warn",
+
+    "arrow-parens": ["error", "as-needed"],
+    "arrow-spacing": "error",
+    "block-spacing": ["error", "always"],
+    "brace-style": "error",
+    "comma-spacing": ["error", { "before": false, "after": true }],
+    "eol-last": ["error", "always"],
+    "func-call-spacing": ["error", "never"],
+    "indent": ["error", 2, { "SwitchCase": 1, "MemberExpression": "off" }],
+    "jsx-quotes": ["error", "prefer-single"],
```

Comment:
I just checked and Studio used more double quotes before. Enabling this rule causes 77 errors. Enabling `prefer-double` only causes 23 errors. Tobira uses double quotes; the editor seems to use double quotes as well. It's also closer to HTML (which JSX wants to mimic). So I'd have preferred to use double quotes. 
Should we still change this or just keep it as is now?